stonesoup/movable/actionable_movable.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
```

Comment:
We might be able to remove the Actionable.register() line in this file anyway as FixedMovable will inherit from MovableActionable (and therefore Actionable).